Added note for trace_id and span_id requirements (#6136)
DataDog/documentation: content/en/api/tracing/send_trace.md

[Learn more about the APM & Distributed Tracing terminology][4]

> **Note**: Each span within a trace should use the same `trace_id`. However, `trace_id` and `span_id` require different values. To avoid unexpected value returns, use different values or a unique scheme.

Reviewer: ```suggestion
**Note**: Each span within a trace should use the same `trace_id`. However, `trace_id` and `span_id` must have different values.
```
